include/zephyr/drivers/cellular.h

```diff
@@ -139,6 +139,8 @@ enum cellular_event {
 	CELLULAR_EVENT_MODEM_INFO_CHANGED = BIT(0),
 	/** Cellular network registration status changed */
 	CELLULAR_EVENT_REGISTRATION_STATUS_CHANGED = BIT(1),
+	/** Result of a communications link check to the modem */
+	CELLULAR_EVENT_MODEM_COMMS_CHECK_RESULT = BIT(2),
```

Comment:
Does the user care if the script was sent successfully? would it not be enough to have `CELLULAR_EVENT_MODEM_COMMS_CHECK_FAILED` and omit the data passed